src/pkg/migrate/heroku.go

```diff
@@ -54,39 +77,21 @@ func collectHerokuApplicationInfo(ctx context.Context, client HerokuClientInterf
 	}
 
 	term.Debugf("Postgres info for the selected application: %+v\n", applicationInfo.PGInfo)
-	dynoSizes := make(map[string]HerokuDynoSize)
-
-	// for each dyno, get the dyno size,
-	for _, dyno := range dynos {
-		dynoSize, err := client.GetDynoSize(ctx, dyno.Size)
-		if err != nil {
-			return HerokuApplicationInfo{}, fmt.Errorf("failed to get dyno size for dyno %s: %w", dyno.Name, err)
-		}
-		dynoSizes[dyno.Name] = dynoSize
-	}
-
-	applicationInfo.DynoSizes = dynoSizes
 
 	configVars, err := client.ListConfigVars(ctx, appName)
 	if err != nil {
 		return HerokuApplicationInfo{}, fmt.Errorf("failed to list Heroku config vars: %w", err)
 	}
 	applicationInfo.ConfigVars = configVars
 
-	releaseTasks, err := client.GetReleaseTasks(ctx, appName)
-	if err != nil {
-		return HerokuApplicationInfo{}, fmt.Errorf("failed to get Heroku release tasks: %w", err)
-	}
-	applicationInfo.ReleaseTasks = releaseTasks
-
 	return applicationInfo, nil
 }
 
 func selectSourceApplication(surveyor surveyor.Surveyor, appNames []string) (string, error) {
 	var selectedApp string
 	for {
 		err := surveyor.AskOne(&survey.Select{
-			Message: "Select the Heroku application to use as a source:",
+			Message: "Select the Heroku application you would like to migrate:",
```

Comment:
`migrate` sounds like we'll *move* it?

Reply:
The prompt is already "Migrate from Heroku" so I'm just keeping the same language.

Reply:
I agree though, that we should have some follow up text that suggests next steps.